gcp/appengine/frontend3/src/styles.scss

```diff
@@ -404,8 +403,21 @@ pre {
   margin-top: 22px;
   border: 2px solid $osv-text-color;
   border-radius: 8px;
-  --mdc-text-field-fill-color: rgba(255, 255, 255, 0.3);
-  --mdc-text-field-label-ink-color: $osv-text-color;
+  --md-filled-text-field-container-color: #696969;
+  --md-filled-text-field-label-text-font: $osv-body-font-family;
+  --md-filled-text-field-input-text-font: $osv-body-font-family;
+  --md-filled-text-field-input-text-weight: 100;
+  --md-filled-text-field-label-text-color: $osv-text-color;
+  --md-filled-text-field-input-text-color: $osv-text-color;
+  --md-filled-text-field-leading-icon-color: $osv-text-color;
+  --md-filled-text-field-focus-label-text-color: $osv-text-color;
+  --md-filled-text-field-focus-input-text-color: $osv-text-color;
+  --md-filled-text-field-focus-leading-icon-color: $osv-text-color;
+  --md-filled-text-field-hover-label-text-color: $osv-text-color;
+  --md-filled-text-field-hover-input-text-color: $osv-text-color;
+  --md-filled-text-field-hover-leading-icon-color: $osv-text-color;
+  --md-filled-text-field-caret-color: $osv-text-color;
+  --md-filled-text-field-focus-caret-color: $osv-text-color;
```

Comment:
Yep, There are many settings here! Another approach is to pass system colours, e.g. `--md-sys-color-on-surface`. I believe setting up this configuration makes it clear what the colour settings are, instead of relying on a few system colours, which will be used as fallbacks if the final colours are not specified. 
However, I'm open to your suggestions.

Reply:
I think I prefer we set the system colours, and add the overrides as needed.

Main reasoning for this is while having a list of all the text colour variables is useful, there's no way to keep it up to date when they add new variables (which I believe they can do without changing the major version), so the list could be misleading, and you would have to look up the variables anyway.

Reply:
Updated via [c7a9bea](https://github.com/google/osv.dev/pull/2314/commits/c7a9bea483be59f447aaaf8fe27bf0051712f093)